src/System.Management.Automation/engine/CommandCompletion/CompletionAnalysis.cs

```diff
@@ -420,6 +420,17 @@ internal List<CompletionResult> GetResultHelper(CompletionContext completionCont
 
                     case TokenKind.StringExpandable:
                     case TokenKind.StringLiteral:
+                        // Search to see if we're looking at an assignment
+                        if (lastAst.Parent is CommandExpressionAst
+                            && lastAst.Parent.Parent is AssignmentStatementAst assignmentAst)
+                        {
+                            // Handle scenarios like '$ErrorActionPreference = '<tab>'
```

Comment:
>  '$ErrorActionPreference = '<tab>'

The single quote here is a little misleading.

Reply:
This code is only hit if there is already a single (or double) quote that needs to be completed.  I can add more to the comment.

Reply:
I understand the scenario we want to address here. I only suggest to change the example a bit, to

> \`$ErrorActionPreference = \'\<tab\>\`